test/kubernetes_unittest.cc

```diff
@@ -68,4 +80,67 @@ TEST_F(KubernetesTest, GetNodeMetadata) {
   });
   EXPECT_EQ(big->ToString(), m.metadata.metadata->ToString());
 }
+
+TEST_F(KubernetesTest, ComputePodAssociations) {
+  const std::string api_version = "1.2.3";
```

Comment:
Let's inline these as well and get rid of the local variables.

Reply:
Done